lib/bike_brigade/messaging/slack.ex

```diff
@@ -0,0 +1,21 @@
+defmodule BikeBrigade.Messaging.Slack do
+  alias BikeBrigade.SlackApi
+
+  defmodule RiderSms do
+    @channel_id "C01QU0YVACW" #dispatch
+
+    def post_message(message) do
+      payload = SlackApi.PayloadBuilder.build(@channel_id, message)
+      :ok = SlackApi.post_message(payload)
+    end
+  end
+
+  defmodule Operations do
+    @channel_id "C016VGHETS4" #software
```

Comment:
Should we set these in the config so we can have a different channel for dev/prod?

Reply:
I like that!

You can just do something like 

```
config :bike_brigade, BikeBrigade.Messaging.Slack.Operations,
   chanel_id: "C016VGHETS4"
   channel_name: "software"
```